crates/rattler_networking/src/authentication_middleware.rs

```diff
@@ -104,6 +137,8 @@ impl AuthenticationMiddleware {
                 }
                 Authentication::CondaToken(_) => Ok(req),
             }
+        } else if req.url().host_str() == Some("storage.googleapis.com") {
```

Comment:
For the `oci` registry support we came up with a new "scheme". I also know that conda has support for an `s3://` scheme. 

Maybe we could also decide that channels that are on Google Cloud should indicate that with a `gcs://storage.googleapis.com/bla`? 

Potentially, people could also have public buckets there (and no need for authentication?).